src/network/networkd-link.c

```diff
@@ -3108,6 +3108,9 @@ static int link_carrier_lost(Link *link) {
 
         assert(link);
 
+        /* Some devices reset itself while setting the MTU. This trigers the DHCP client fall into a LOOP.
```

Comment:
trigers -> triggers

Reply:
Also you cannot use "trigger" with "into".

Reply:
what should be the correct way then ?